### PR TITLE
[MM-748] Add client_max_body_size

### PR DESCRIPTION
**Pull request by Inlustra**

### CODE REVIEW

**Review by jamesthimont — COMMENTED** [review]
Is the value overwritten anywhere? Where is it set to 1mb?

**Comment by Inlustra** [comment]
@jamesthimont 1mb is the default for nginx, it's not overwritten, it's just never set! 
https://stackoverflow.com/questions/28476643/default-nginx-client-max-body-size

File: scripts/config/templates/nginx.conf.erb [hunk 1]
```diff
@@ -50,6 +50,8 @@ http {
     resolver <%= resolver %>;
   <% end %>
 
+    client_max_body_size 32m;
```
Comment (AzinKhan) [inline scripts/config/templates/nginx.conf.erb]: Should this be in the higher up http block? My understanding is that nested blocks inherit from above but blocks with nested ones within don't inherit from their children. If we put it in http {} then I think it's global, which might be better.

Comment (Inlustra) [reply scripts/config/templates/nginx.conf.erb]: I put it in the parent block! :)

**Review by jamesthimont — APPROVED** [review]

**Pull request merged**